CHANGES.rst

```diff
@@ -1,6 +1,12 @@
 Changes
 =======
 
+Version 0.8.1 (UNRELEASED)
+--------------------------
+
+- Administrators:
+    - Adds new configuration option, ``reana_server.environment.REANA_SCHEDULER_REQUEUE_COUNT``, to set requeue count for submitted workflows in case of scheduling error.
```

Comment:
Very minor: I think it should start with `Adds configuration environment variable` like with other envars (e.g. `REANA_SCHEDULER_REQUEUE_SLEEP`)